bin/render.cpp

```diff
@@ -53,6 +53,12 @@ int main(int argc, char *argv[]) {
         exit(2);
     }
 
+    if (backendValue) {
```

Comment:
nit: this code is repeated several times, it would make sense to encapsulate it within a function, maybe inside `util` namespace.